### ISSUE TITLE
Home page

### ISSUE THREAD

**Issue by wayferer** [modelrecords/modelrecords#82]
The UMR home page should be inspired by 
- https://paperswithcode.com/
- https://mlcommons.org/about-us/

Content outline (to be refined @sschafft ) 
- [x] github link somewhere
- [x] Title: Unified Model Records
- [x] Leader: Tracking models and their artifacts in a systematic, versionable, and open source repository.
- [x] TL; DR: Foundation models depend on high quality curated datasets. UMRs enable you to track model and dataset provenance and receive alerts when upstream changes are discovered.
- [x] Summary of the paper: @asahahaha  (which should motivate why UMRs are good idea)
- [x] Create your Unified Model Record
-- generates LaTeX, PDF, HTML and GraphViz format
-- CLI tool how to use (with a code block and 5 step start)

Notable Cards
- [ ] hardcoded list using same styling as all cards page
- [ ] filter by card type: family, model, dataset and system
- [ ] link to all Cards

Cards should be organized by card_type:
* family:
https://github.com/modelrecords/modelrecords/blob/main/modelrecords/repository/llama2/llama2-1.0.0.yaml#L2
* model:
https://github.com/modelrecords/modelrecords/blob/main/modelrecords/repository/llama2-13b/llama2-13b-1.0.0.yaml#L188
* dataset:
https://github.com/modelrecords/modelrecords/blob/main/modelrecords/repository/laion-5b/laion-5b-1.0.0.yaml#L2
* system:
ChatGPT, or the breast cancer prediction system (@asahahaha)

How to cite us (see examples like the openai/clip readme)
- [ ] Instructions

**Comment by wayferer** [comment]
Styling is done. Need to add copy

**Comment by wayferer** [comment]
Summary from @asahahaha :

Model cards and system cards are considered the gold standard for pro- viding model-to-model comparisons between various foundation models. These tools enable good documentation for the system under question but often fall short of tracing the model genealogy, machine readability, providing reliable centralized management systems and fostering incentives for their creation. For example, tracing the provenance of foundation models and their derivatives is essential to managing licensing and legislative risks in the AI ecosystem, as it allows researchers to understand a model’s entire history and dependencies. It is also well known that such documents like model cards on HuggingFace are inconsistently produced in non-machine-readable formats, often with erroneous information or missing details. We argue these are symptoms of a deeper issue - that creating model cards or system cards is a manual process where human effort is duplicated and error-prone, notably when a new model inherits most of its design from a foundation model. 
To close this gap, we propose a machine-readable model specification format which can flexibly capture model information. Importantly, this format explicitly delineates the relationships between upstream and downstream models, as well as their associated data. To facilitate the adoption, we have released the Unified Model Record (UMR) repository, which serves as a semantically versioned repository of model specifications, enabling better traceability and transparency in the AI, while being user-friendly, thereby motivating machine learning practitioners to maintain documentation.

**Comment by wayferer** [comment]
TL;DR: A standardized record for Foundation Models, and their upstream and downstream dependencies.
